### DISCUSSION TITLE
Do you need to build custom Docker image for ingesting assets?

### DISCUSSION THREAD

**Discussion by muety** [carla-simulator/carla#6715] [Q&A]
The [docs](https://carla.readthedocs.io/en/latest/tuto_A_add_props/#ingestion-in-a-carla-package) mention that, in order to build a package for custom assets (e.g. vehicle or map), you'll have to build a Docker image of CARLA / UE4 first. Since this takes forever and uses such a lot of disk space, I'd rather try to avoid having to do this. In my understanding, that image doesn't depend in your custom assets at all, but is just used for _building_ then. Isn't there a pre-built image published somewhere already?

**Comment by muety** [comment]
Isn't there a way to just quickly [build a standalone asset package](https://carla.readthedocs.io/en/latest/tuto_A_create_standalone/) (given an `.fbx` file), without having to build - among others - the UE4 **Editor** first?

![image](https://github.com/carla-simulator/carla/assets/8978843/74a709fd-a846-4d93-bc0b-f36ac2b08716)

**Reply by shubhamKGIT** [reply]
Hi Muety,

Did you find a solution to this. My docker build is failing from some compatibility issue with UE4. It's in step make CarlaUE4Editor. Any leads would be helpful. 

Regards and thanks.
